backend/base/templates/bookings_list.html

```diff
@@ -1,123 +1,4 @@
-            id="navbarDropdown"
-            href="#"
-            role="button"
-            data-bs-toggle="dropdown"
-            aria-expanded="false"
-          >
-            <i class="fas fa-user fa-fw"></i>
-            {{ user.username }}
-          </a>
-          <ul
-            class="dropdown-menu dropdown-menu-end"
-            aria-labelledby="navbarDropdown"
-          >
-            <!-- <li><a class="dropdown-item" href="#!">Settings</a></li>
-                        <li><a class="dropdown-item" href="#!">Activity Log</a></li>
-                        <li><hr class="dropdown-divider" /></li> -->
-            <li>
-              <a class="dropdown-item" href="{% url 'logout' %}">Logout</a>
-            </li>
-          </ul>
-        </li>
-      </ul>
-    </nav>
-    <div id="layoutSidenav">
-      <div id="layoutSidenav_nav">
-        <nav class="sb-sidenav accordion sb-sidenav-dark" id="sidenavAccordion">
-          <div class="sb-sidenav-menu">
-            <div class="nav">
-              <div class="sb-sidenav-menu-heading">Organization</div>
-              <a class="nav-link" href="{% url 'tenantuser_page' %}">
-                <div class="sb-nav-link-icon">
-                  <i class="fas fa-tachometer-alt"></i>
-                </div>
-                Dashboard
-              </a>
-              <a
-                class="nav-link"
-                href="{% url 'booking_list'  %}"
-                style="color: white; background-color: rgb(68, 68, 68)"
-              >
-                <div class="sb-nav-link-icon">
-                  <i class="fas fa-calendar"></i>
-                </div>
-                Bookings
-              </a>
-              <a class="nav-link" href="{% url 'tenant_customers_list'%}">
-                <div class="sb-nav-link-icon"><i class="fas fa-users"></i></div>
-                Customers
-              </a>
-              <div class="sb-sidenav-menu-heading">Approvals</div>
-              <a class="nav-link" href="{% url 'organization_list'  %}">
-                <div class="sb-nav-link-icon">
-                  <i class="fas fa-building"></i>
-                </div>
-                Organizations
-              </a>
-              <a class="nav-link" href="{% url 'location_list'  %}">
-                <div class="sb-nav-link-icon">
-                  <i class="fas fa-map-marker-alt"></i>
-                </div>
-                Locations
-              </a>
-            </div>
-          </div>
-          <!-- Your HTML template -->
-          <div class="sb-sidenav-footer">
-            <div class="small">Logged in as:</div>
-            <div class="user-email">{{ request.user.email }}</div>
-          </div>
-        </nav>
-      </div>
-      <div id="layoutSidenav_content">
-        <main>
+{%  extends 'base_tenant.html' %} {% block content %}
           <div class="container-fluid px-4">
             <h1 class="mt-4 mb-4" style="margin-left: 40%">Booking List</h1>
```

Comment:
Remove commented lines and dont use inline style. add it to external css created.